src/teams.json

```diff
@@ -133,7 +133,15 @@
         "googleapis/gapic-generator",
         "googleapis/nodejs-proto-files",
         "googleapis/proto-plus-python",
-        "googleapis/sample-tester"
+        "googleapis/sample-tester",
+        "googleapis/google-auth-library-swift",
+        "googleapis/google-api-swift-client",
+        "googleapis/api-common-protos",
+        "googleapis/googleapis.github.io",
+        "googleapis/kiosk",
+        "googleapis/gax-kotlin",
+        "googleapis/disco-to-proto3-converter",
+        "googleapis/api-linter"
```

Comment:
Sorry to be sniping this PR at the last second, but is it possible to assign this to the `api-linter` team? `actools` doesn't actually own this project.

Reply:
No worries at all - this is why I'm leaving it open!  On assigning things to the api-linter team specifically - not quite yet :/  I could create a separate team in sloth for them, give them their own dashboard and such, but it would require them using yoshi live instead of buganizer (for now) as the source of truth.  

Not sure what to do with these quite yet.

Reply:
Ok, no problem, I didn't realize this team assignment was more than just a GitHub team :) Assign away!

Reply:
We do not really use Buganizer for linter bugs anyway (we post to GitHub).